dotcom-rendering/src/components/CardPicture.tsx

```diff
@@ -82,19 +84,26 @@ const block = css`
 	display: block;
 `;
 
-/** On fronts, all images are 5:3 */
-const aspectRatio = css`
-	padding-top: ${(3 / 5) * 100}%;
-	position: relative;
-
-	& > * {
-		position: absolute;
-		top: 0;
-		left: 0;
-		width: 100%;
-		height: 100%;
-	}
-`;
+/**
+ * On fronts, Fairground cards have an image ration of 5:4.
+ * This is due to replace the existing card ratio of 5:3
+ * For now, we are keeping both ratios.
+ */
+const decideAspectRatio = (aspectRatio: AspectRatio) => {
```

Comment:
Wonder if we can give a slightly different name for the function or prop? It feels a bit weird to see something 'deciding aspect ratio' with an aspect ratio prop. 

Maybe... eh, even something like `decideAspectRatioStyle` or something?

Reply:
good point, perhaps decideAspectRatioPadding?